src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -64,16 +64,18 @@
     private Map<String, Object> opts;
 
     /**
-     * A simple map of the most recently issued token for any given stream.
+     * A simple map of the most recently issued global token for any given stream.
      */
-    ConcurrentHashMap<UUID, Long> lastIssuedMap;
+    ConcurrentHashMap<UUID, Long> lastGlobalTokenMap;
 
-    // A map of stream local addresses
+    /**
+     * A simple map of the most recently issued local offset for any given stream.
```

Comment:
I am nit picking, but is there a difference between 'token' (used above) and 'offset' (used here)?